Orion Express courier shuttle tweaks

hazelrat opened the pull request:
A few very small changes and fixes to the OE ship, primarily focused on making the shuttle more functional.

The shuttle has been reworked to have functional power, a more interesting appearance, more resources available, and multiple areas. The intention is that a large volume of cargo can be easily loaded onto it. Previously, the cabling was broken in addition to all the flooring being shuttle floors, which cannot apparently be pried, so it was essentially impossible to fix.  Associated code could use a glance-over.

![image](https://github.com/Aurorastation/Aurora.3/assets/83198434/49a0db6c-6721-42ab-8e15-18750e0dabf4)

APCs throughout the ship have been set to OE ship access, so the crew can actually access them now.

hazelrat commented:
!review

DreamySkrell reviewed (approved):
![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/e6d43208-5364-48e2-aa1a-50c91e3114de)

would be nice if the big OX logo I added to the OX automated station, was added somewhere to the OX courier station and/or shuttle as well

hazelrat commented:
> ![image](https://private-user-images.githubusercontent.com/107256943/301197834-e6d43208-5364-48e2-aa1a-50c91e3114de.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY3MTU3MjUsIm5iZiI6MTcwNjcxNTQyNSwicGF0aCI6Ii8xMDcyNTY5NDMvMzAxMTk3ODM0LWU2ZDQzMjA4LTUzNjQtNDhlMi1hYTFhLTUwYzkxZTMxMTRkZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEzMVQxNTM3MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ODhjNDMyNGE4NzA2YjBlYTEzZTQzZWQ1MWRhNDYzZjZmOTFlOWE1Yjg1MmUyMzYwMmU2NDFkODhkZjY5ZDlmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.FtflipkNoZZ1VYYagLZw97OSwNTh_ZVcSbPY6jXycOE)
> 
> would be nice if the big OX logo I added to the OX automated station, was added somewhere to the OX courier station and/or shuttle as well

Implemented. Had to shuffle some furniture around for it, and neatened up some unusual piping in the meantime.
![image](https://github.com/Aurorastation/Aurora.3/assets/83198434/beb8930d-dbc9-4516-8a85-61ba7cf23889)
